README.md

```diff
@@ -184,4 +189,4 @@ I would like to thank the Microverse full-stack curriculum for the inspiration a
 
```

Comment:
![image](https://github.com/Lul-Abdifan/Blog-App/assets/93039359/03c71a13-d4a6-43c5-ad57-65892d82fa67)

- [ ] It would be nice if you fix the Live Link which you have provided in the Live Demo section because it is broken right now 😨 If you don't wanna deploy your app right now then kindly remove the whole Live Demo section and whenever you will have a working live link then you can add this section again to make your readme descriptive and professional 👍